app/dashboard/models.py

```diff
@@ -2478,3 +2479,37 @@ def __str__(self):
     def coinbase_short(self):
         coinbase_short = f"{self.coinbase[0:5]}...{self.coinbase[-4:]}"
         return coinbase_short
+
+
+class HackathonEvent(SuperModel):
+    """A hackathon event"""
+
+    name = models.CharField(max_length=255)
+    logo = models.FileField(blank=True)
```

Comment:
Can you switch this to `models.ImageField` ?

Reply:
@mbeacom I actually did first, turns out it doesn't like SVGs.

Reply:
Yeah.  If we expect SVGs, then it'll have to be FileField. 🙌 but if that's the case, we'll want to handle the svg -> png if we anticipate sending out the branding in an email.

Reply:
Doesn't base64 encoding the svg and embedding that into the email do the trick?
Not well informed about email clients 😄

Reply:
If the email service supports displaying SVGs, sure. But many still don't whether it's base64 encoded or not.

Reply:
@mbeacom
Thanks for the update 👍